### PR TITLE
[Accessibility] Contact modal form accessibility corrections

### PR DESCRIPTION
**Pull request by Destinyyyy**
### Description
- Contact form corrections
- Updated @codegouv/react-dsfr version to include changes related to accessibility
  - https://github.com/codegouvfr/react-dsfr/releases/tag/v1.13.6
### Ticket
- https://www.notion.so/Audit-avec-tickets-notion-526ecd6d84764c0c84844c2e41071fe2?p=89cd7f76b2854e49b011c4341ea76afd&pm=s

### CODE REVIEW

File: site/src/app/v2/une-question/components/ContactForm/ContactForm.tsx [hunk 1]
```diff
@@ -230,9 +228,8 @@ const ContactForm = ({ closeFn, isProVersion }: Props) => {
                     name: 'lastname',
                     onChange: (e: ChangeEvent<HTMLInputElement>) =>
                       onInputChanged(e.target.value, 'lastname'),
-                    'aria-label': 'Saisir votre nom',
                     autoComplete: 'family-name',
-                    'aria-autocomplete': 'none',
+                    required: true,
```
Comment (P3rceval) [inline site/src/app/v2/une-question/components/ContactForm/ContactForm.tsx:232]: ça va rentrer en conflit avec ma PR du coup => https://github.com/betagouv/pass-sport/pull/322/files

Comment (P3rceval) [reply site/src/app/v2/une-question/components/ContactForm/ContactForm.tsx:232]: Mais il en manque un chez toi xD

Comment (Destinyyyy) [reply site/src/app/v2/une-question/components/ContactForm/ContactForm.tsx:232]: Il manque lequel du coup?

Comment (Destinyyyy) [reply site/src/app/v2/une-question/components/ContactForm/ContactForm.tsx:232]: https://github.com/betagouv/pass-sport/pull/320/commits/366eefee807efd76463d5498ed7b833641a6d505

**Pull request merged**